### PR TITLE
feat: Add an s2i tip for not supporting containerd runtime

### PR DESCRIPTION
**Pull request by weili520**
Signed-off-by: TheYoungManLi <cjl@kubesphere.io>

### What type of PR is this?
/kind feature

### Which issue(s) this PR fixes:
Fixes ##[2714](https://github.com/kubesphere/console/issues/2714)

### Special notes for reviewers:

截屏2021-12-03 10 37 47

### Does this PR introduced a user-facing change?
```release-note
Add an S2i tip for not supporting containerd runtime.
```

### CODE REVIEW

**Comment by zryfish** [comment]
`containerd` is the word, I don't think I've ever seen `ContainerD`.

https://containerd.io/

**Comment by weili520** [comment]
> `containerd` is the word, I don't think I've ever seen `ContainerD`.
> 
> https://containerd.io/

ok
截屏2021-12-03 10 46 16

File: server/services/session.js [hunk 1]
```diff
@@ -246,6 +246,26 @@ const getKSConfig = async token => {
   return resp
 }
 
+const getK8sRuntime = async ctx => {
+  const token = ctx.cookies.get('token')
+  let resp = {}
+  try {
+    const nodeList = await Promise.all([
```
Comment (xuliwenwenwen) [inline server/services/session.js:253]: Can be `await send_gateway_request({...})`

Comment (weili520) [reply server/services/session.js]: done

File: locales/zh/s2i.js [hunk 2]
```diff
@@ -65,6 +65,8 @@ module.exports = {
   IMAGE_FROM_B2I: '通过制品构建镜像',
   S2I_DESC: '选择代码的开发语言。',
   B2I_DESC: '选择制品的文件类型。',
+  CONTAINERD_RUNTIME_NOT_SUPPORT:
+    's2i 和 b2i 暂时不支持 containerd 容器运行时。',
```
Comment (JohnNiang) [inline locales/zh/s2i.js:69]: Please @kubesphere/sig-docs  help us review these entry.

Comment (Felixnoo) [reply locales/zh/s2i.js:69]: @JohnNiang   I think we need to change `s2i` to `S2I` and `b2i` to `B2I`?

BTW, please put the EN verison `S2I and B2I do not support the containerd runtime.` to the file `locales/en/l10n-projects-imageBuilders-list.js`.
/cc @Patrick-LuoYu

Comment (JohnNiang) [reply locales/zh/s2i.js:69]: +1 Make sense to me.

Comment (weili520) [reply locales/zh/s2i.js]: Done

**Review by JohnNiang — APPROVED** [review]
I confirm that this approach works well.


- For containerd container runtime

image
image

- For docker container runtime

image
image

/lgtm

**Pull request merged**